.github/workflows/build.yml

```diff
@@ -28,6 +28,9 @@ jobs:
         arch: [x86, x86_64]
         build_type: [Debug, Release]
         portable: [Portable, Non-Portable]
+        exclude:
+          - build_type: Debug
+            portable: Portable
```

Comment:
Is there any reason to do it differently here and on lines 84/85, here we are excluding, but there we just removed from matrix. I have seen users request a windows portable build in the past (mjt), to which I would direct them to the actions, Can we justify it's removal?

Reply:
Lines 84/85 is removing _all_ {portable or debug} builds from WinXP, we only need to perform standard release builds to ensure it works. Anyone wanting a portable or debug build for WinXP are gonna have to do it themselves.

Here, we're only excluding the windows build if it's debug _and_ portable. So we'll get {release, portable} and {debug + non-portable} but not {debug, portable}.

So to be clear, we will provide portable Windows builds. Just not for WinXP.

Reply:
Okay, that makes sense, thanks for clearing that up. In that case, LGTM.